.github/workflows/cross-version.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [21, 22]
+        java: [22, 23]
```

Comment:
I don't think there is any early-access build for 23, I think we keep 22 only for now.

Reply:
Yea, I checked after the pipeline failed. Removed 23 :)